reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md

```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
+command is enclosed in parentheses so that the **PathsWithInstances** property returns each path
+instance as a string.
 
-The third command uses the **Get-Counter** cmdlet to get the counter values. It uses the **Counter** parameter to specify the counters and the **ComputerName** parameter to specify the computers saved in the $servers variable.
-PS C:\> Get-Counter -Counter $Counter -ComputerName $Servers
-```
+### Example 10: Get a single value for each counter in a counter set
 
-These commands get the value of the "**% DPC Time**" performance counter on 50 randomly select computers in the enterprise.
-### Example 8
-```
-The first command uses the **Get-Counter** cmdlet to get the counter paths. It saves them in the $MemCounters variable.
-PS C:\> $MemCounters = (Get-Counter -List Memory).Paths
+A single value is returned for each performance counter in the local computer's **Memory** counter
+set.
 
-The second command uses the **Get-Counter** cmdlet to get the counter data for each counter. It uses the **Counter** parameter to specify the counters in the $MemCounters variable.
-PS C:\> Get-Counter -Counter $MemCounters
+```powershell
+$MemCounters = (Get-Counter -ListSet Memory).Paths
+Get-Counter -Counter $MemCounters
 ```
 
-These commands get a single value for all of the performance counters in the Memory counter set on the local computer.
-### Example 9
+```Output
+Timestamp                 CounterSamples
+---------                 --------------
+6/19/2019 12:05:00        \\Computer01\memory\page faults/sec :
+                          868.772077545597
+
+                          \\Computer01\memory\available bytes :
+                          9031176192
+
+                          \\Computer01\memory\committed bytes :
+                          8242982912
+
+                          \\Computer01\memory\commit limit :
+                          19603333120
 ```
-The first command saves a counter path in the $Counter variable.
-PS C:\> $Counter = "\\SERVER01\Process(Idle)\% Processor Time"
 
-The second command uses the **Get-Counter** cmdlet to get one sample of the counter values. It saves the results in the $Data variable.
-PS C:\> $Data = Get-Counter $Counter
+`Get-Counter` uses the **ListSet** parameter to specify the **Memory** counter set. The command is
+enclosed in parentheses so that the **Paths** property returns each path as a string. The paths are
+stored in the `$MemCounters` variable. `Get-Counter` uses the **Counter** parameter to specify the
+counter paths in the `$MemCounters` variable.
+
+### Example 11: Display an object's property values
 
-The third command uses the Format-List cmdlet to display all the properties of the **CounterSamples** property of the sample set object as a list.
-PS C:\> $Data.CounterSamples | Format-List -Property *
+The property values in the **PerformanceCounterSample** object represent each data sample. You can
```

Comment:
The property values in the **PerformanceCounterSample** object represent each data sample. In this example we use the properties of the **CounterSamples** object to examine, select, sort, and group the data.

Reply:
resolved.